cmd/tsbs_load_cratedb/creator.go

```diff
@@ -0,0 +1,219 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"github.com/jackc/pgx"
+	"log"
+	"strings"
+)
+
+type tableDef struct {
+	schema string
+	name   string
+	tags   []string
+	cols   []string
+}
+
+func (t *tableDef) fqn() string {
```

Comment:
```
// fqn returns the fully-qualified name of a table
```

took me a bit to figure out what `fqn` stood for, so maybe a comment will help others 😄

Reply:
godoc comments start with the name of the function, so it should start with `// fqn returns...`